spec/classes/failover_spec.rb

```diff
@@ -15,11 +15,12 @@
           :peer_address => '10.1.1.20',
         } end
 
-        let(:facts) do {
-          :concat_basedir => '/doesnotexist',
-          :domain         => 'example.org',
-          :osfamily       => osfamily,
-        } end
+        let(:facts) do
+          facts.merge({
+            :concat_basedir => '/doesnotexist',
+            :domain         => 'example.org',
```

Comment:
The default facts also have a domain which we can use. IMHO there's no need to override it.